releng/raw-binaries/fs/usr/local/bin/influxdb_raw_binaries.bash

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+if [ -z "$GOOS" ] || [ -z "$GOARCH" ]; then
+  >&2 echo 'The environment variables $GOOS and $GOARCH must both be set.'
+  exit 1
+fi
+
+# Extract tarball into GOPATH.
+tar xz -C "$GOPATH" -f /influxdb-src.tar.gz
+
+SHA=$(jq -r .sha < "$GOPATH/src/github.com/influxdata/influxdb/.metadata.json")
+
+OUTDIR=$(mktemp -d)
+for cmd in \
+  influxdb/cmd/influxd \
+  influxdb/cmd/influx_stress \
+  influxdb/cmd/influx \
+  influxdb/cmd/influx_inspect \
+  influxdb/cmd/influx_tsm \
+  ; do
+    go build -i -o "$OUTDIR/$(basename $cmd)" "github.com/influxdata/$cmd"
+done
+
+SUFFIX=
+if [ "$CGO_ENABLED" == "0" ]; then
+  # Only add the static suffix to the filename when explicitly requested.
+  SUFFIX=_static
+fi
```

Comment:
Needs support for race-enabled builds here. I'm fine either way of including it in this PR or keeping it a separate PR to prevent this PR from growing further.